sensor/kubernetes/localscanner/service_certificates_repository.go

```diff
@@ -69,6 +69,24 @@ func newServiceCertificatesRepo(ownerReference metav1.OwnerReference, namespace
 				serviceCertFileName: mtls.ServiceCertFileName,
 				serviceKeyFileName:  mtls.ServiceKeyFileName,
 			},
+			storage.ServiceType_SCANNER_V4_INDEXER_SERVICE: {
```

Comment:
Do I understand correctly that we're going to use the same certificate for all scanner components v2 and v4 combined?

Reply:
No, different certificates for all scanner components.